src/ucs/memory/numa_defs.h

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) NVIDIA CORPORATION & AFFILIATES, 2023. ALL RIGHTS RESERVED.
+ *
+ * See file LICENSE for terms.
+ */
+
+#ifndef UCS_NUMA_DEFS_H_
+#define UCS_NUMA_DEFS_H_
+
+#include <stdint.h>
+
+#define UCS_NUMA_MIN_DISTANCE 10
+
+typedef int ucs_numa_distance_t;
+typedef uint16_t ucs_numa_node_t;
+
+
+/**
+ * @return The number of CPU cores in the system.
+ */
+int ucs_numa_num_configured_cpus();
+
```

Comment:
2 space lines in H file

Reply:
@ofirfarjun7 unresolved

Reply:
I removed this file

Reply:
You moved the issue to `numa.h` but it didn't disappear.